extension/bearertokenauthextension/bearertokenauth_test.go

```diff
@@ -54,9 +53,6 @@ func TestBearerAuthenticator(t *testing.T) {
 	assert.NoError(t, err)
 	assert.NotNil(t, credential)
 
-	_, ok := credential.(credentials.PerRPCCredentials)
-	assert.True(t, ok)
```

Comment:
Why is this deleted?

Reply:
because the return `credential` is of a type `credentials.PerRPCCredentials` so this check is useless, always will be that, we nolonger return an interface so no check is needed it is ensured by the compiler.